pages/services/cassandra/2.6.0-3.11.4/configuration/index.md

```diff
@@ -0,0 +1,281 @@
+
+*   **In DC/OS CLI options.json**: `cpus`: number (default: `0.5`)
+*   **DC/OS web interface**: `CASSANDRA_CPUS`: `number`
+
+### Memory
+
+You can customize the amount of RAM allocated to each node. Change this value by editing the **mem** value (in MB) under the **node** configuration section. Setting this too low will result in out of memory errors. The `heap.size` setting must also be less than this value to prevent out of memory errors, which can result when the Java Virtual Machine attempts to allocate more memory than is available to the {{ model.techShortName }} process.
+
+*   **In DC/OS CLI options.json**: `mem`: integer (default: `10240`)
+*   **DC/OS web interface**: `CASSANDRA_MEMORY_MB`: `integer`
+
+### JMX Port
+
+You can customize the port that {{ model.techName }} listens on for JMX requests, such as those issued by `nodetool`.
+
+*   **In DC/OS CLI options.json**: `jmx_port`: integer (default: `7199`)
+*   **DC/OS web interface**: `TASKCFG_ALL_JMX_PORT`: `integer`
+
+### Storage Port
+
+You can customize the port that {{ model.techName }} listens on for inter-node communication.
+
+*   **In DC/OS CLI options.json**: `storage_port`: integer (default: `7000`)
+*   **DC/OS web interface**: `TASKCFG_ALL_CASSANDRA_STORAGE_PORT`: `integer`
+
+### SSL Storage Port
+
+You can customize the port that {{ model.techName }} listens on for inter-node communication over SSL.
+
+*   **In DC/OS CLI options.json**: `ssl_storage_port`: integer (default: `7001`)
+*   **DC/OS web interface**: `TASKCFG_ALL_CASSANDRA_SSL_STORAGE_PORT`: `integer`
+
+### Native Transport Port
+
+You can customize the port that {{ model.techName }} listens on for CQL queries.
+
+*   **In DC/OS CLI options.json**: `native_transport_port`: integer (default: `9042`)
+*   **DC/OS web interface**: `TASKCFG_ALL_CASSANDRA_NATIVE_TRANSPORT_PORT`: `integer`
+
+### RPC Port
+
+You can customize the port that {{ model.techName }} listens on for Thrift RPC requests.
+
+*   **In DC/OS CLI options.json**: `rpc_port`: integer (default: `9160`)
+*   **DC/OS web interface**: `TASKCFG_ALL_CASSANDRA_RPC_PORT`: `integer`
+
+### Native Authentication and Authorization
+
+To use Cassandra's native `PasswordAuthenticator` and `CassandraAuthorizer`, set the following configs:
+
+```
+{
+  "service": {
+    "security": {
+      "authentication": {
+        "enabled": true,
+        "superuser": {
+          "name": "<superuser-name>",
+          "password_secret_path": "<service-name>/password"
+        }
+      },
+      "authorization": {
+        "enabled": true
+      }
+    }
+  }
+}
+
+```
+
+This will deploy your Cassandra cluster with enhanced security protection which includes disabling the native cassandra superuser and replacing it with the superuser you specify.
```

Comment:
```suggestion
This will deploy your Cassandra cluster with enhanced security protection, which includes disabling the native cassandra superuser and replacing it with the superuser that you specify.
```